taskcluster/ci/signing/kind.yml

```diff
@@ -62,7 +66,7 @@ job-template:
           default: autograph_apk
     index:
         by-build-type:
-            (fennec-.+|nightly|performance-test|beta|production|debug):
+            (fennec-.+|nightly|performance-test|beta|production|debug|nightly-simulation):
```

Comment:
Based on Callek's input, I think `nightly-simulation` can be a temporary solution while we implement what Gecko has. Do you think we need to expose the `nightly-simulation` builds to https://firefox-ci-tc.services.mozilla.com/tasks/index/mobile.v2.fenix ? If so, please keep this change. If not, you revert this line.

I don't have any strong opinion. I leave the final call to you @gmierz 🙂

Reply:
I checked with some other people in CI about this and it seems like there's no easy way of doing that - active data will be the simplest solution for this. So, I'll remove this change :)

Reply:
Chatted on Matrix. We want to keep this index for the purpose of being able to query builds made on a given day.